README.md

```diff
@@ -1,2 +1,38 @@
-# Task-List
- iOS Bootcamp Homework Module 5
+## Part 1 - Short answer questions.
+
+a) **Modifiers** are used to customize the appearance and behavior of a View.
+
+b) The basic building block of a SwiftUI interface is a **View**.
+
+c) Each View in SwiftUI is a **value** type.
+
+d) List two ways you can find the modifiers of a View.
+- **Dot syntax: type a `.` after the view and let autocomplete show you the options**
+- **The modifiers tab in the Library (lists modifiers by category)**
+
+e) True or False: A modifier changes a View.</br>
+**False, each modifier returns a new view.**
+
+f) Modifiers are performed and returned in **top-to-bottom order** from the **view** they are applied to. 
+
+g) When the data driving a View changes, that View has to be **re-rendered** to reflect the change.
+
+h) True or False: The @Binding establishes a two-way communication between Views.</br>
+**False, `@Binding` establishes a two-way connection between a view and its *data*.**
```

Comment:
False. Use a binding to create a two-way connection between a property that stores data, and a view that displays and changes the data.